dev/prod/src/platform.ts

```diff
@@ -152,6 +152,7 @@ export async function configurePlatform() {
   addLocation(documentId, () => import(/* webpackChunkName: "document" */ '@hcengineering/document-resources'))
   addLocation(bitrixId, () => import(/* webpackChunkName: "bitrix" */ '@hcengineering/bitrix-resources'))
   addLocation(requestId, () => import(/* webpackChunkName: "request" */ '@hcengineering/request-resources'))
+  addLocation(presentationId, () => import(/* webpackChunkName: "presentation" */ '@hcengineering/presentation/src/resources'))
```

Comment:
Why we need it?

Reply:
I understand, but we probable could be without it, if generic approach for aggregation will be selected.